meshroom/ui/graph.py

```diff
@@ -283,6 +284,9 @@ def setGraph(self, g):
         """ Set the internal graph. """
         if self._graph:
             self.stopExecution()
+            # Clear all the submitted nodes at once before the graph gets changed, as it won't receive further updates
+            if self._computingLocally:
+                self._graph.clearSubmittedNodes()
```

Comment:
It will change the status of all nodes including those submitted externally in renderfarm.

Reply:
You're right, nodes submitted externally would also have been cleared if at least one chunk was being computed locally at the same time. This is now fixed!